### PR TITLE
Catch fixture error

### PR DESCRIPTION
**Pull request by benrudolph**
This is a much better error message for the user when a fixture is not found. Realized how bad this was when debugging: http://manage.dimagi.com/default.asp?179662#999451

Top one is old message, bottom one is the new message
screen shot 2015-08-21 at 4 37 19 pm

### CODE REVIEW

**Comment by benrudolph** [comment]
@czue buddy: @biyeun

File: touchforms/backend/touchcare.py [hunk 1]
```diff
@@ -348,7 +348,10 @@ def _get_fixture(self, user_id, fixture_id):
                                                         "user": user_id,
                                                         "fixture": fixture_id }
         q = query_factory(self.vars.get('host'), self.vars['domain'], self.auth, format="raw")
-        results = q(query_url)
+        try:
+            results = q(query_url)
+        except (HTTPError, URLError), e:
+            raise TouchFormsNotFound('Unable to fetch fixture at %s: %s' % (query_url, str(e)))
```
Comment (czue) [inline touchforms/backend/touchcare.py]: should be marked for translation

Comment (czue) [inline touchforms/backend/touchcare.py]: oh wait nevermind, didn't see was the in jython process

Comment (benrudolph) [inline touchforms/backend/touchcare.py]: :disappointed:

**Pull request merged**